project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/.github/workflows/docgen_from_Jira.yaml

```diff
@@ -0,0 +1,61 @@
+name: docgen from Jira
+on:
+  workflow_dispatch
+
+jobs:
+   generate:
+     runs-on: ubuntu-latest
+     steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # full history for metadata
+          submodules: false
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Python install
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install "lander<2.0.0"
+
+      - name: condaenv
+        run: |
+          conda init bash
+          conda create --name docsteady-env docsteady -c lsst-dm -c conda-forge
+        shell: bash
+
+      - name: docugen
+        env:
+          JIRA_USER: {% raw %}${{ secrets.JIRA_USER }} {% endraw %}
+          JIRA_PASSWORD: {% raw %}${{ secrets.JIRA_PASSWORD }} {% endraw %}
+        run: |
+          source ${CONDA}/bin/activate docsteady-env
+          docsteady generate-tpr {{ cookiecutter.plan }}  {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex
```

Comment:
For non-DM reports, I think this needs to be:
```
docsteady --namespace {{ cookiecutter.namespace }} generate-tpr ...
```

Reply:
This is passed to the tex template - if it were a tech note it would put System Engineering or DM on the front page. Since its a test report the front page does not carry a subsystem and it has no effect. I tried SE instead of DM for SCTR-14 and it fails .. I would rather take a separate ticket and remove this option since it has no effect ulness you show me what it does ...

Reply:
I was going from the documentation but did not check the actual template.  No problem if it doesn't actually have an effect.  Perhaps it used to but it was changed?

Reply:
Looking more - PSE is the correct namsepace for SE in tex it expects SE not PSE .. so its not consistent. It is also not in the namespace i.e. Gabriele did not bother to collect it in sqrbot-jnr..

Reply:
In the first version the front page was probalby not modifed and carried the subsystem name .

Reply:
crud found it .. in the abstract .. ok I will  add namesapce to the template and the action